Core/Utilities/TimeLeft/TimeLeft.py

```diff
@@ -76,10 +85,23 @@ def getTimeLeft( self, cpuConsumed = 0.0 ):
       return resourceDict
 
     resources = resourceDict['Value']
-    if not resources['CPULimit'] or not resources['WallClockLimit']:
+    self.log.verbose( resources )
```

Comment:
Please avoid this kind of logging without any context... It makes debugging impossible... Also when logging a full result dictionary it would be better to set it to "debug" level as this is really very low level ("debug" has a level lower than "verbose")